assets/js/googlesitekit/datastore/user/permissions.test.js

```diff
@@ -92,6 +93,82 @@ describe( 'core/user authentication', () => {
 				).toEqual( someError );
 			} );
 		} );
+
+		describe( 'refreshCapabilities', () => {
+			it( 'updates capabilities from server', async () => {
+				const updatedCapabilities = {
+					googlesitekit_view_dashboard: true,
+					googlesitekit_manage_options: true,
+					'googlesitekit_read_shared_module_data::["site-verification"]': true,
+					'googlesitekit_read_shared_module_data::["tagmanager"]': true,
+					'googlesitekit_read_shared_module_data::["optimize"]': false,
+					'googlesitekit_read_shared_module_data::["adsense"]': false,
+					'googlesitekit_manage_module_sharing_options::["search-console"]': true,
+					'googlesitekit_read_shared_module_data::["search-console"]': false,
+					'googlesitekit_read_shared_module_data::["analytics"]': false,
+					'googlesitekit_read_shared_module_data::["pagespeed-insights"]': false,
+					'googlesitekit_read_shared_module_data::["idea-hub"]': false,
+				};
+				global[ capabilitiesBaseVar ] = capabilities;
+				expect(
+					registry.select( CORE_USER ).getCapabilities()
+				).toEqual( capabilities.permissions );
+
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/core\/user\/data\/permissions/,
+					{
+						body: updatedCapabilities,
+						status: 200,
+					}
+				);
+
+				await registry.dispatch( CORE_USER ).refreshCapabilities();
+
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+				expect(
+					registry.select( CORE_USER ).getCapabilities()
+				).toEqual( updatedCapabilities );
+			} );
+
+			it( 'Sets permissionScopeError when api throws an error', async () => {
```

Comment:
The casing in this description is slightly inconsistent with elsewhere:
```suggestion
			it( 'sets permissionScopeError when API throws an error', async () => {
```